py3status/modules/diskdata.py

```diff
@@ -79,6 +87,39 @@ class Py3status:
     }
     unit = "B/s"
 
+    class Meta:
+        deprecated = {
+            "remove": [{"param": "sector_size", "msg": "obsolete parameter"}],
+            "rename": [
+                {"param": "disk", "new": "disks", "msg": "rename parameter to `disks`"}
+            ],
+        }
+
+    def _resolve_disk_name(self, path):
+        disk_path = os.path.realpath(path)
+        return os.path.basename(disk_path)
+
+    def _get_sector_size(self, drive):
+        basename = self._resolve_disk_name(drive)
+        block_device = os.path.realpath("/sys/class/block/" + basename)
+        if not block_device.endswith("block/" + basename):
+            block_device = os.path.dirname(block_device)
+        with open("{}/queue/hw_sector_size".format(block_device)) as ss:
+            out = int(ss.read().strip())
+        return out
+
+    def _add_monitored_disk(self, path):
+        basename = self._resolve_disk_name(path)
+        if os.path.exists("/dev/" + basename):
+            self._disks.add(Disk(device="/dev/" + basename, name=basename))
+
+    def _get_all_disks(self):
```

Comment:
(Fluffy) Something else other than `_get` --> None.  Maybe `set`, `add`, or `monitor`?

Reply:
Can you please explain? I'm not getting the point

Reply:
Sorry. `_get_all_disks` is slightly misleading (i.e. it does not get anything; return `None`). I wanted to know if we can use something else... such as `set_all_disks` or `add_all_disks` or `monitor_all_disks`. Cheers.

Reply:
You're right, I've refactored it